ex04.ex

```diff
@@ -39,7 +39,7 @@ defmodule Ex04 do
       [ 1, 2, 3, 4, 5 ]
 
   """
-  def reverse . . . "your code"
+  def reverse(xs), do: reduce(xs,[],&([&1|&2]))
```

Comment:
No need for extra parens:

~~~ elixir
def reverse(xs), do: reduce(xs, [], &[&1|&2])
~~~